pyuvdata/uvdata/uvdata.py

```diff
@@ -897,7 +892,11 @@ def check(
             )
         if np.any(
             np.isclose(
-                np.linalg.norm(self.uvw_array[~autos], axis=1),
+                np.sqrt(
+                    self.uvw_array[~autos, 0] ** 2
+                    + self.uvw_array[~autos, 1] ** 2
+                    + self.uvw_array[~autos, 2] ** 2
+                ),
```

Comment:
Turns out doing this square and square root is faster than `numpy.linalg.norm` but I can change it back if that is desired, or more readable.